src/providers/ColorSchemeProvider.tsx

```diff
@@ -1,10 +1,11 @@
 import { useAsyncStorage } from '@react-native-async-storage/async-storage'
-import { FC, useEffect } from 'react'
+import { useColorMode } from 'native-base'
```

Comment:
We should not need this file as we have `colorModeManager` https://docs.nativebase.io/next/color-mode#h2-persisting-the-color-mode